src/@types/reactProps.d.ts

```diff
@@ -3,6 +3,8 @@ type BoardProps = {
   handlePlayerInput: (coords: Coords) => void;
   reset: () => void;
   winner?: string;
+  winningCells?: number[][] | undefined;
```

Comment:
I'm not convinced that `Vectors` lends any more meaning than `number[][]` does, but if you decide to keep it you should probably use it here, too.

Reply:
^^

Reply:
cosigned!

Reply:
Turns out we really want this to be an array of tuples, so: even better!